recipes/chi2comb-py/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "chi2comb-py" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/chi2comb/chi2comb-{{ version }}.tar.gz
+  sha256: 5e209d8fe0a1b4df2e3c48af61146903f3ca38a73b2a50672f43fc98ee9c6adf
+
+build:
+  number: 0
+  skip: true  # [py27]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cffi >=1.11.5
+    - chi2comb >=0.0.3
+    - pip
+    - pytest-runner >=4.2
+    - python
+    - setuptools >=38.4.0
+  run:
+    - pytest-doctestplus >=0.3.0
+    - cffi >=1.11.5
+    - chi2comb >=0.0.3
+    - pytest >=3.6.3
+    - pytest-runner >=4.2
+    - python
+
+test:
+  requires:
```

Comment:
`pytest` is not required for `imports` tests. Be aware that these are the test-requirements of the build process, not the test requirements of the package.